source/trading/Permissions.rst

```diff
@@ -0,0 +1,63 @@
+.. links:
+.. _`/groups`: https://www.tradingview.com/rest-api-spec/#operation/getGroups
+.. _`/permissions`: https://www.tradingview.com/rest-api-spec/#operation/getPermissions
+
+.. _permissions-endpoint:
+
+Permissions
+-------------
+
+You can restrict access to market data or hide symbols for some users.
+There are several cases when it might be needed:
+
+- Symbols can be hidden depending on the users' location. For example, users from Spain will be able to see symbols that are hidden from Italian users.
+- Symbols can be hidden from the *Symbol Search* bar. However, any user can open a chart by entering a full symbol name (i.e., EXCHANGE:SYM1SYM2).
+- Symbols can only be displayed after users log into their brokerage account.
+- If the broker uses data provided by TradingView, the broker's users may also need to pay for the TradingView subscription to get real-time data. This may happen in the following cases:
+
+  - The broker doesn't cover the real-time data subscription for users.
+  - The real-time data provision is not subject to local compliance requirements.
+
+  To prevent users from paying twice: on the broker's and TradingView's ends, TradingView can provide real-time data to verified integration users.
```

Comment:
```suggestion
  To prevent users from paying twice: on the broker's and TradingView's ends, TradingView can provide real&#8209time data to verified integration users.
```